backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/LabelWidget.java

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.forms;
+
+import java.text.MessageFormat;
+import java.util.List;
+import java.util.Objects;
+
+import org.eclipse.sirius.components.annotations.Immutable;
+import org.eclipse.sirius.components.forms.validation.Diagnostic;
+
+/**
+ * The label widget.
+ *
+ * @author fbarbin
+ */
+@Immutable
+public final class LabelWidget extends AbstractWidget {
```

Comment:
Why this class and some others are named after the pattern `LabelWidget` but some others are named after the pattern `Label`?
Unless I'm missing something, I would have preferred consistency here and named all related class after the pattern `Label`.
(backend and frontend)

Reply:
Because of a conflict in the graphql API with the type Label (already used in diagram

Reply:
I use Label for the DSL view. And LabelWidget for the technical part (graphql api, typescript types )